docs/docs/contribute/contributing-guide.mdx

```diff
@@ -0,0 +1,57 @@
+# Contributing to the CLI for Microsoft 365
+
+We love to accept contributions. If you want to get involved in helping us grow the CLI for Microsoft 365, whether that is suggesting or adding a new command, extending an existing command, or updating our documentation, we would love to hear from you.
+
+Before you submit your first PR, please read the following guide. We'd hate to see you work on something that someone else is already working on, something that we agreed not to do, or something that doesn't match the project.
+
+Sharing is caring!
+
+## Introduction to CLI for Microsoft 365
```

Comment:
Do we need this section? I would assume that if you are thinking about contributing to the CLI you will already know what it is so seems unnecessary to repeat it here.